### PR TITLE
Conclude `osu! Amateur Solo Indonesia Showdown`

### PR DESCRIPTION
**Pull request by Nivalyx**
- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/de.md [hunk 1]
```diff
@@ -214,6 +214,12 @@ Nicht offizielle Turniere/Wettkämpfe, die von den Communities veranstaltet werd
 | :-- | :-- | :-- | :-- | :-- | :-- |
 | [Old Map Fantasy World Cup](OMF/WC) | 14.08.2021 | 03.10.2021 | ::{ flag=US }:: Vereinigte Staaten | ::{ flag=KR }:: Südkorea | ::{ flag=PL }:: Polen |
 
+#### osu! Amateur Solo Indonesia Showdown
+
+| Name | Start | Ende | ![Goldkrone][GCrown] | ![Silberkrone][SCrown] | ![Bronzekrone][BCrown] |
+| :-- | :-- | :-- | :-- | :-- | :-- |
+| [osu! Amateur Solo Indonesia Showdown](oASIS/1) | 2023-02-06 | 2023-03-21 | ::{ flag=ID }:: [im a cute bunny](https://osu.ppy.sh/users/12578000) | ::{ flag=ID }:: [SDKO](https://osu.ppy.sh/users/4858555) | ::{ flag=ID }:: [akebono](https://osu.ppy.sh/users/1815316) |
```
Comment (Walavouchey) [inline wiki/Tournaments/de.md:221]:
```suggestion
| [osu! Amateur Solo Indonesia Showdown](oASIS/1) | 06.02.2023 | 21.03.2023 | ::{ flag=ID }:: [im a cute bunny](https://osu.ppy.sh/users/12578000) | ::{ flag=ID }:: [SDKO](https://osu.ppy.sh/users/4858555) | ::{ flag=ID }:: [akebono](https://osu.ppy.sh/users/1815316) |
```

File: wiki/Tournaments/fr.md [hunk 2]
```diff
@@ -214,6 +214,12 @@ Tournois et compétitions non-officiels gérés par la communauté.
 | :-- | :-- | :-- | :-- | :-- | :-- |
 | [Old Map Fantasy World Cup](OMF/WC) | 14/08/2021 | 03/10/2021 | ::{ flag=US }:: États-Unis | ::{ flag=KR }:: Corée du Sud | ::{ flag=PL }:: Pologne |
 
+#### osu! Amateur Solo Indonesia Showdown
+
+| Nom | Début | Fin | ![Couronne en or][GCrown] | ![Couronne d'argent][SCrown] | ![Couronne de bronze][BCrown] |
+| :-- | :-- | :-- | :-- | :-- | :-- |
+| [osu! Amateur Solo Indonesia Showdown](oASIS/1) | 2023-02-06 | 2023-03-21 | ::{ flag=ID }:: [im a cute bunny](https://osu.ppy.sh/users/12578000) | ::{ flag=ID }:: [SDKO](https://osu.ppy.sh/users/4858555) | ::{ flag=ID }:: [akebono](https://osu.ppy.sh/users/1815316) |
```
Comment (Walavouchey) [inline wiki/Tournaments/fr.md:221]:
```suggestion
| [osu! Amateur Solo Indonesia Showdown](oASIS/1) |  06/02/2023 | 21/03/2023 | ::{ flag=ID }:: [im a cute bunny](https://osu.ppy.sh/users/12578000) | ::{ flag=ID }:: [SDKO](https://osu.ppy.sh/users/4858555) | ::{ flag=ID }:: [akebono](https://osu.ppy.sh/users/1815316) |
```

File: wiki/Tournaments/oASIS/1/en.md [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
   - oASIS 2023
+  - o!ASIS 2023
 ---
 
 # osu! Amateur Solo Indonesia Showdown
 
 ![osu! Amateur Solo Indonesia Showdown logo](img/logo.jpg)
 
-The **osu! Amateur Solo Indonesia Showdown** (***oASIS***) is a double-elimination 1v1 osu! tournament hosted by ::{ flag=ID }:: [Millie](https://osu.ppy.sh/users/3484548). The tournament is open to all players from Indonesia with a global rank of #10,000 or lower. It is the first instalment of the osu! Amateur Solo Indonesia Showdown tournament series.
+The **osu! Amateur Solo Indonesia Showdown** (***oASIS/o!ASIS***) was a double-elimination 1v1 osu! tournament hosted by ::{ flag=ID }:: [Millie](https://osu.ppy.sh/users/3484548). The tournament was open to all players from Indonesia with a global rank of #10,000 or lower. It was the first instalment of the osu! Amateur Solo Indonesia Showdown tournament series.
```
Comment (Walavouchey) [inline wiki/Tournaments/oASIS/1/en.md:13]: or maybe just pick one instead?

```suggestion
The **osu! Amateur Solo Indonesia Showdown** (***oASIS*** or ***o!ASIS***) was a double-elimination 1v1 osu! tournament hosted by ::{ flag=ID }:: [Millie](https://osu.ppy.sh/users/3484548). The tournament was open to all players from Indonesia with a global rank of #10,000 or lower. It was the first instalment of the osu! Amateur Solo Indonesia Showdown tournament series.
```

File: wiki/Tournaments/oASIS/1/img/podium.jpg
Comment (Walavouchey) [inline wiki/Tournaments/oASIS/1/img/podium.jpg:1]: this image lol

**Review by Nivalyx — COMMENTED** [review]
e i clicked the wrong button and made this comment by accident lol

please ignore this

File: wiki/Tournaments/oASIS/1/img/podium.jpg
Comment (Nivalyx) [reply wiki/Tournaments/oASIS/1/img/podium.jpg:1]: ya that's a running joke that goes across the tournament lol

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**